### PR TITLE
Made WinUI 2.x optional on uwp

### LINKED ISSUE #58: MultiTarget: Add option to disable WinUI 2.x on UWP
### Describe the bug

Found in https://github.com/CommunityToolkit/Labs-Windows/pull/438

> @Arlodotexe I think your infra is injecting extra dependencies into packages, look at this:
> 
> 
> 
> I'm not WinUI nor the `Immutable` package anywhere, and they shouldn't be there. Could you take a look? 🙂

This component does not use WinUI in any way, but targets UWP. This should be added to facilitate this scenario in AppServices and other future components.

### Steps to reproduce

```text
n/a
```

### Expected behavior

n/a

### Screenshots

_No response_

### Code Platform

- [ ] UWP
- [ ] WinAppSDK / WinUI 3
- [ ] Web Assembly (WASM)
- [ ] Android
- [ ] iOS
- [ ] MacOS
- [ ] Linux / GTK

### Windows Build Number

- [ ] Windows 10 1809 (Build 17763)
- [ ] Windows 10 1903 (Build 18362)
- [ ] Windows 10 1909 (Build 18363)
- [ ] Windows 10 2004 (Build 19041)
- [ ] Windows 10 20H2 (Build 19042)
- [ ] Windows 10 21H1 (Build 19043)
- [ ] Windows 11 21H2 (Build 22000)
- [ ] Other (specify)

### Other Windows Build number

_No response_

### App minimum and target SDK version

- [ ] Windows 10, version 1809 (Build 17763)
- [ ] Windows 10, version 1903 (Build 18362)
- [ ] Windows 10, version 1909 (Build 18363)
- [ ] Windows 10, version 2004 (Build 19041)
- [ ] Other (specify)

### Other SDK version

_No response_

### Visual Studio Version

_No response_

### Visual Studio Build Number

_No response_

### Device form factor

_No response_

### Additional context

_No response_

### Help us help you

Yes, I'd like to be assigned to work on this item.

### PR DESCRIPTION
**Pull request by Arlodotexe**
Closes #58 

This PR makes use of the existing `HasWinUI` property. When in a library, this property can be set to 'false' to override the default. On UWP, this will prevent `Microsoft.Xaml.UI` from being included.

### CODE REVIEW

File: MultiTarget/MultiTargetIdentifiers.props [hunk 1]
```diff
@@ -46,17 +46,17 @@
     <IsDroid Condition="'$(IsDroid)' == '' AND '$(TargetFramework)' == '$(AndroidLibTargetFramework)' AND '$(AndroidLibTargetFramework)' != '' AND '$(MultiTargetsDroid)' == 'true'">true</IsDroid>
     <IsMacOS Condition="'$(IsMacOS)' == '' AND '$(TargetFramework)' == '$(MacOSLibTargetFramework)' AND '$(MacOSLibTargetFramework)' != '' AND '$(MultiTargetsMacOS)' == 'true'">true</IsMacOS>
     <IsiOS Condition="'$(IsiOS)' == '' AND '$(TargetFramework)' == '$(iOSLibTargetFramework)' AND '$(iOSLibTargetFramework)' != '' AND '$(MultiTargetsiOS)' == 'true'">true</IsiOS>
-    <IsNetstandard Condition="'$(IsNetstandard)' == '' AND $(MultiTargetsNetstandard) == 'true'">true</IsNetstandard>
+    <IsNetstandard Condition="'$(IsNetstandard)' == '' AND '$(TargetFramework)' == '$(NetStandardCommonTargetFramework)' AND '$(MultiTargetsNetstandard)' == 'true'">true</IsNetstandard>
```
Comment (Sergio0694) [inline MultiTarget/MultiTargetIdentifiers.props:49]: Can you explain this change? Seems unrelated to WinUI? Is this needed to make that work? 🤔

File: MultiTarget/MultiTargetIdentifiers.props [hunk 2]
```diff
@@ -46,17 +46,17 @@
...
 
     <IsUno Condition="'$(IsWasm)' == 'true' OR '$(IsWpf)' == 'true' OR '$(IsGtk)' == 'true' OR '$(IsDroid)' == 'true' OR '$(IsMacOS)' == 'true' OR '$(IsiOS)' == 'true'">true</IsUno>
 
-    <HasWinUI Condition="'$(IsUwp)' == 'true' OR '$(IsWinAppSdk)' == 'true' OR '$(IsUno)' == 'true'">true</HasWinUI>
+    <HasWinUI Condition="'$(HasWinUI)' == '' AND '$(IsUwp)' == 'true' OR '$(IsWinAppSdk)' == 'true' OR '$(IsUno)' == 'true'">true</HasWinUI>
```
Comment (Sergio0694) [inline MultiTarget/MultiTargetIdentifiers.props:53]: Might want to add parentheses around the AND operands to make the priority clearer.

**Review by Sergio0694 — APPROVED** [review]
Looks good! Just left a couple nits

File: MultiTarget/MultiTargetIdentifiers.props (hunk 1 above)
Comment (Arlodotexe) [reply MultiTarget/MultiTargetIdentifiers.props:49]: Not needed to make this work, just a bonus hotfix since we saw this broken when using in AppServices
